internal/pkg/template/templates/addons/s3/env/cf.yml

```diff
@@ -0,0 +1,54 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The name of the environment being deployed.
```

Comment:
Maybe just
```suggestion
    Description: Your environment's name.
```
🤔

Reply:
I am debating internally, probably to an unnecessary extent 😅  I wanted to frame this description so that it implies that people can write, for example, `Mappings` depending on the value of `Env`. "Your environment's name" makes it sound like the env addon template is going to be deployed with one specific env; while in reality, it will get deployed with any envs... I might be overthinking this way too much. But do you think this concern make sense?

Reply:
In the workload version, the description to the `Name` parameter is actually `Description: The name of the service, job, or workflow being deployed.` instead of "Your workload's name". Although I think it makes sense in this case to describe it as "Your workload's name" because a workload addon is indeed associated with a specific workload in most cases.

Anyway, I can change it to "Your environment's name".

Reply:
I like it as is. 🤷🏼‍♀️

Reply:
I will keep it as is!